examples/run_examples.sh

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-
-# Azure 
```

Comment:
ah fuck me we are gonna need to squash this can't be in the commit history

Reply:
yeah I don't know where this got added

Reply:
definetly my fault